### ISSUE TITLE
show error when images do not load in captcha

### ISSUE THREAD

**Issue by goastler** [prosopo/captcha#252]

**Comment by goastler** [comment]
It's jarring for the user to see the image alt text when the image doesn't load. This isn't something we want the user to see or be confused about, as they could proceed with the captcha and fail it because of this. Therefore, we should detect when an image has not loaded and throw an error, making the captcha process restart from the beginning with a new provider with working images. Later, we can report this event to begin slashing providers whose images do not load

**Comment by goastler** [comment]
![Image](https://github.com/prosopo/captcha/assets/7059456/45ab888a-6aa1-4670-aa14-c94ac7308614)

**Comment by HughParry** [comment]
it should error out and query a new provider

**Comment by goastler** [comment]
images loaded via the <img> tag can detect when the img fails to load, e.g. 404. However, it may be easier to fetch the images ourselves and then manage the result when it fails, especially if we're triggering a new provider query

**Comment by HughParry** [comment]
What are the desired outcomes here? Refine!

**Comment by HughParry** [comment]
@HughParry To drop ticket and spec out new reload frontend button ticket with AC similar to frictionless reload
